front/context/src/AccountsContext.tsx

```diff
@@ -36,17 +36,23 @@ export function AccountsContextProvider(props: Props): React.ReactElement {
    * Fetch accounts from the extension
    */
   async function fetchAccounts(): Promise<void> {
-    const extensions = await web3Enable(originName);
-
-    if (!extensions.length) {
-      throw new Error(
-        'No extension found. Please install PolkadotJS extension.'
+    if (typeof window !== `undefined`) {
```

Comment:
there are multiple instances of this, we could put a constant `IS_SSR=window !== `undefined``